service/libs/commonutils/utilities.go

```diff
@@ -24,3 +28,41 @@ func DecodeJSONWithHresult(r io.Reader, v interface{}) error {
 	}
 	return nil
 }
+
+func DumpStacks(baseLogPath string, writeToFile bool) {
```

Comment:
Since this is a guest and its different than the host. We don't need `writeToFile` support and we should return this as part of the RPC rather than using `stdout` propagation.